### ISSUE TITLE
The documentation - starting with Model > Probability Distributions > Show Model

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#3248]
The mosaic qdist command seems to be given OK in the example from our instructions, but there is no graph.  I tried changing the location for the graph, because that is all we had before, but still there is no graph.  It isn't a ggplot graph.

**Comment by rdstern** [comment]
Now I have tried it again.  I now do get the graph.

But I still get it in the separate window, even when other graphs are coming in the output window.  In our documentation we say that this will be "fixed in the future".  Is it easy to do now?

I repeat it isn't a ggplot.  I don't know if that makes any difference.

**Comment by dannyparsons** [comment]
I think because the graph is just displayed and not returned in R by this function we can't store it, or prevent it popping up in a separate window.

**Comment by rdstern** [comment]
OK, so this is a "feature", and not a bug.

I am going to leave this then and we can later perhaps discuss with the mosaic people?

**Comment by dannyparsons** [comment]
Yes because it's doing two things, the values and the graph. Note in R if you did:
`x <- mosaic::qdist(mean=0, sd=1, plot=TRUE, dist="norm", p=c(0.5))`
It would still display the graph and `x` would contain the values only. There is no way to store the graph or prevent it's display. The issue is that the function does 2 things, so only one is returned and the graph is just printed. This should be two separate functions really so both could be stored. It doesn't need to be a ggplot graph to do this.

**Comment by dannyparsons** [comment]
Yes we could discuss with `mosaic` because although I understand why they've done it, it's not the standard R way. Basically, this is an issue in R so we don't need to solve it ourselves.

**Comment by lilyclements** [comment]
Closing because "this is an issue in R so we don't need to solve it ourselves.", and also because our graphics system on displaying plots has come much further since 2017
